data/json/mapgen/dairy_farm.json

```diff
@@ -0,0 +1,251 @@
+[
+    {
+    "name": "GROUP_COWS",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "is_safe": true,
+    "monsters": [
+      { "monster": "mon_cow", "freq": 100, "cost_multiplier": 10, "ends": 168 },
+      { "monster": "mon_cow", "freq": 50, "cost_multiplier": 5, "starts": 336 }
+    ]
+   {,
+  {
```

Comment:
I'm surprised the linter is letting this one go -- there's an open brace, comma, and misaligned new open brace here. Maybe it's just github being weird with display.